pages/en/lb4/Testing-your-application.md

```diff
@@ -42,19 +43,15 @@ integrates these packages and makes them easy to use together with LoopBack.
 
 ### Set up testing infrastructure with LoopBack CLI
 
-{% include note.html content="The LoopBack CLI does not yet support LoopBack 4,
-so using the CLI is not an option with the beta release.
-" %}
-
-<!--
-If you are just getting started with LoopBack, use the LoopBack command-line tool (CLI)  `loopback-cli`. It's ready to use and installs simply by `npm install -g loopback-cli`. You don't need to do any extra steps for setup, and can head straight to the [next section](Testing-Your-Application#acceptance-testing.html).
--->
+LoopBack applications that have been generated by `lb4 app` or `lb4`
```

Comment:
You can add here that if the user selects all our add-ons such as `tslint` and `mocha` during app creation, then they get them added as deps too with configs.